packages/darklang/prettyPrinter/packages.dark

```diff
@@ -10,7 +10,9 @@ module Darklang =
         { name: String
           types: List<PACKAGE.Darklang.LanguageTools.ProgramTypes.PackageType.T>
           fns: List<PACKAGE.Darklang.LanguageTools.ProgramTypes.PackageFn.T>
-          submodules: List<PACKAGE.Darklang.PrettyPrinter.Packages.Module> }
+          submodules: List<PACKAGE.Darklang.PrettyPrinter.Packages.Module>
+          constants : List<PACKAGE.Darklang.LanguageTools.ProgramTypes.PackageConstant.T>
```

Comment:
bit of a nitpick: I think submodules should be printed either first or last, and that these fields should match up with that.